modin/core/execution/dask/common/task_wrapper.py

```diff
@@ -39,11 +41,14 @@ def deploy(cls, func, num_returns, kwargs):
             The result of ``func`` splitted into parts in accordance with ``num_returns``.
         """
         client = default_client()
-        remote_task_future = client.submit(func, **kwargs)
-        return [
-            client.submit(lambda l, i: l[i], remote_task_future, i)
-            for i in range(num_returns)
-        ]
+        remote_task_future = client.submit(func, *args, **kwargs)
+        pure = kwargs.get("pure", None)
```

Comment:
Why is a default value None? Given distributed's docs, pure is equal to True by default.

Reply:
I saw it in [the code](http://distributed.dask.org/en/stable/_modules/distributed/client.html#Client.submit).

I think it's beneficial in our case to make the default value - `False`, since that's what is used in most cases.